mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/dependencies/DependencyResolver.java

```diff
@@ -6,12 +6,15 @@
 import org.apache.maven.model.Model;
 import org.apache.maven.model.Plugin;
 import org.arquillian.smart.testing.Configuration;
+import org.arquillian.smart.testing.Logger;
 import org.arquillian.smart.testing.mvn.ext.ApplicablePlugins;
 
 public class DependencyResolver {
 
     private final Configuration configuration;
 
+    private Logger logger = Logger.getLogger(DependencyResolver.class);
```

Comment:
Why do we need this logger here if we don't log anything from this class anymore?